examples/telegraf/zookeeper/zookeeper_data_size_batch.tick

```diff
@@ -0,0 +1,47 @@
+// zookeeper_data_size_batch
+
+// metric: "approximate_data_size"
+// available_fields: "avg_latency","ephemerals_count","max_file_descriptor_count","max_latency","min_latency","num_alive_connections","open_file_descriptor_count","outstanding_requests","packets_received","packets_sent","version","watch_count","znode_count" 
+
+// TELEGRAF CONFIGURATION
+// [inputs.zookeeper]
+//   servers = [":2181"]
+
+// DEFINE: kapacitor define zookeeper_data_size_batch -type batch -tick zookeeper/zookeeper_data_size_batch.tick -dbrp telegraf.autogen
+// ENABLE: kapacitor enable zookeeper_data_size_batch
+
+// Parameters
+var info = 100000000 // 100MB
```

Comment:
We should make this easier to use. Possibly by defining a sizeUnit var and then multiplying the thresholds by it. For example

```
var mb = 1024*1024 // 1 MB
var gb = 1024*mb // 1 GB

var info = 100 * mb 
var warn = 1 * gb 
var crit = 10 * gb
```

It demonstrates some of the flexibility of TICKscript and it reads better to me. Thoughts?